recipes/powerlaw/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "powerlaw" %}
+{% set version = "1.4.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "7f9d9cb39ad23af8f6cae165d47020ee8b7fa58f380c356de91ae2625577699e"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
```

Comment:
It looks like `host` only requires `pip` and `python` as the rest are not required to build the package

Reply:
Removed from the host